scripts/master_node/download_images.sh

```diff
@@ -24,37 +24,36 @@ else
 fi
 
 ARCH=$(uname -m)
-VERSION="${VERSION:-latest}"
+VERSION="latest"
+PIP=pip.conf.bak
 
 echo_b "Downloading the docker images for Cello services: VERSION=${VERSION} ARCH=${ARCH}"
 
-# TODO: will be removed after we have the user dashboard image
-echo_b "Check node:9.2 image."
-[ -z "$(docker images -q node:9.2 2> /dev/null)" ] && { echo "pulling node:9.2"; docker pull node:9.2; }
-
 # docker image
 
-for IMG in dashboard nginx api-engine; do
+for IMG in dashboard api-engine; do
 	HLC_IMG=hyperledger/cello-${IMG}
-	#if [ -z "$(docker images -q ${HLC_IMG}:${ARCH}-${VERSION} 2> /dev/null)" ]; then  # not exist
-	echo_b "Pulling ${HLC_IMG}:${ARCH}-${VERSION} from dockerhub"
-	docker pull ${HLC_IMG}:${ARCH}-${VERSION}
-	docker tag ${HLC_IMG}:${ARCH}-${VERSION} ${HLC_IMG}  # match the docker-compose file
-	#else
-	#	echo_g "${HLC_IMG} already exist locally"
-	#fi
+	PATH=build_image/docker/common/${IMG}/Dockerfile.in
+	if [ -z "$(docker images -q ${HLC_IMG}:latest 2> /dev/null)" ]; then  # not exist
+	echo_b "Build ${HLC_IMG} locally"
+	docker build -t ${HLC_IMG}:latest -f ${PATH} ./ --build-arg pip=${PIP} --platform linux/${ARCH}
```

Comment:
Build dashboard and api engine images locally

Reply:
Need to create a new script, e.g., build_images.sh

Otherwise, quite confusing to do build work inside a download_image.sh